test/noInputPrefixRule.spec.ts

```diff
@@ -36,6 +36,25 @@ describe(ruleName, () => {
       });
     });
 
+    it('should fail when an input property is strictly equal to a blacklisted prefix', () => {
+      const prefixes = ['should'];
+      const propertyName = prefixes[0];
+      const inputExpression = `@Input() ${propertyName}: boolean;`;
+      const source = `
+        @Directive()
+        class ${className} {
+          ${inputExpression}
```

Comment:
Let's not use interpolation that aggressively. If we duplicate `should` twice in a single test case, this is fine. Currently, the code is a bit hard to follow.

Reply:
Right. Changing...

Reply:
Done. I hope it's clear now :)